code/modules/clothing/under/ties.dm

```diff
@@ -84,6 +84,7 @@
 	desc = "A bronze medal."
 	icon_state = "bronze"
 	item_color = "bronze"
+	materials = list(MAT_METAL=250)

```

Comment:
re: ergo -- the latter is intended on my end, gives R&D something to do other than wank with their tech.

the former I will fix